### PR TITLE
Add snip-cli

### PR DESCRIPTION
**Pull request by slavovojacek**
Adds [snip-cli](https://github.com/sniptt-official/snip-cli) to the list

#### New App Submission

- [x] I've read the [contribution guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md#readme).

**Repo or homepage link:**

https://github.com/sniptt-official/snip-cli

**Description:**

Sniptt is a developer-friendly secret manager.

It simplifies and improves the experience of sharing end-to-end encrypted secrets such as API keys, database passwords, and other credentials between engineers.

You can finally say goodbye to 1Password, LastPass, and Dashlane 👋.

Read more in this introductory post on [Sharing secrets with teams](https://sniptt.medium.com/sharing-secrets-with-teams-82fde5b11149).

**Why I think it's awesome:**

Top 3 reasons would be

1. Fast and slick onboarding; The onboarding experience is better than some of the most widely used SaaS alternatives out there. It’s dead simple, no BS 😎.

render1623001190591

2. Ease of use and flexibility; The CLI does few things, and it does them well. It is intuitive, interactive, but also flexible enough that it can support more complicated use cases such as scripting and automation.

render1623001219589

render1623002362551

3. Low barrier to entry; Prompts, colours, helpers such as storing master passphrase in OS keychain, etc.

In addition, there is pretty solid documentation, different install options (Homebrew, direct binary install for macOS and Linux, and npm), and [great FAQs](https://github.com/sniptt-official/snip-cli#faq) explaining how stuff works under the hood.

In terms of features to look out for, the one-time secret functionality is pretty cool, and very useful when one wants to quickly share something securely with a recipient and make sure that the secret is destroyed once viewed. Try `snip share` to see it in action.

render1623011550584

### CODE REVIEW

**Comment by jneidel** [comment]
> It simplifies and improves the experience of sharing end-to-end encrypted secrets such as API keys, database passwords, and other credentials between engineers.

This is not grammatically correct. `end-to-end encrypted secrets` describes the secrets (noun) as E2E encrypted (adjective) , which does not make any sense. What you are wanting to say is that the secrets being transmitted using E2E encyption. To correct the sentense:

> It simplifies and improves the experience of *end-to-end encryptedly* sharing  secrets such as API keys, database passwords, and other credentials between engineers.

`encrypted` (adjective) -> `encryptedly` (adverb) now it describes the process of `sharing`, which is what you wanted to say. [It's doesn't sound good now though](https://ell.stackexchange.com/questions/243348/whats-the-adverb-to-unencrypted).

---

Apart from that I disagree with your narrative. You want to replace slack and password managers for sharing passwords securely.
But why not just use Signal, or another secure messenger (not WA or Telegram), instead? You should be using these anyways.

---

> The onboarding experience is better than some of the most widely used SaaS alternatives out there.

Easy to throw that stuff around if you don't even say what you're comparing yourself to.

---

I get that the cli does other stuff, but the tagline 

> Goodbye password managers, hello Sniptt

just sound all kinds of wrong. You are trying to replace a usage of password managers I've never heard of and has nothing with the normal use case of password managers ([normal as in how they describe themselves](https://keepassxc.org/))

---

In conclusion: none of what I've seen here inspires confidence.
